bin/node/cli/src/service.rs

```diff
@@ -507,12 +509,12 @@ pub fn new_full_base(
 	}
 
 	network_starter.start_network();
-	Ok(NewFullBase { task_manager, client, network, transaction_pool })
+	Ok(NewFullBase { task_manager, client, network, transaction_pool, rpc_handlers })
 }
 
 /// Builds a new service for a full client.
-pub fn new_full(config: Configuration) -> Result<TaskManager, ServiceError> {
-	new_full_base(config, |_, _| ()).map(|NewFullBase { task_manager, .. }| task_manager)
+pub fn new_full(config: Configuration) -> Result<(TaskManager, RpcHandlers), ServiceError> {
+	new_full_base(config, |_, _| ()).map(|NewFullBase { task_manager, rpc_handlers, .. }| (task_manager, rpc_handlers))
 }
```

Comment:
You don't need this change, you can just call `new_full_base`

Reply:
Thanks, fixed